tests/build.snapshot.js

```diff
@@ -7,7 +7,8 @@ const smallBuildCommons = {
 	'polyfills.js': { size: 4620 },
 	'polyfills.js.map': { size: 31760 },
 	'favicon.ico': { size: 15086 },
-	'sw.js': { size: 3330 },
+	'sw.js': { size: 1628 },
+	'workbox-sw.prod.v1.1.0.js': {size: 47026},
```

Comment:
The serviceworker filename needs to be `sw.js`, otherwise we'll decapitate any site built with the current version of preact-cli.

Also what's the deal with the filesize?  Did the serviceworker code really go from 1.6kb to 47kb?